iOS/Flows/Home/_New/new_ConversationCollectionViewLayout.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  new_ConversationCollectionViewLayout.swift
+//  Jibber
+//
+//  Created by Martin Young on 9/20/21.
+//  Copyright © 2021 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+
+class new_ConversationCollectionViewLayout: UICollectionViewFlowLayout {
```

Comment:
You can see above I deleted the compositional layout approach I was taking and started replacing it with a traditional flow layout. Not a lot in here at the moment though.